src/menu/flower.cljs

```diff
@@ -60,12 +60,13 @@
                                     2))
          height (j/cell= (/ radius 12))
          ; rotated-offset (j/cell= (* 2 width))
-         left (j/cell= (+ radius (- (/ width 2))))
-         top (j/cell= (+ radius (- (/ height 2))))
+         left (j/cell= (.round js/Math (+ radius (- (/ width 2)))))
```

Comment:
you can just do `Math/round` btw, and this works in both java and javascript (because JS copied the java Math object)